vnext/Scripts/Microsoft.ReactNative.ProjectReunion.nuspec

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.ReactNative.ProjectReunion</id>
+    <projectUrl>https://github.com/microsoft/react-native-windows</projectUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <license type="expression">MIT</license>
+    <copyright>&#169; Microsoft Corporation. All rights reserved.</copyright>
+    <repository type="git"
+      url="https://github.com/microsoft/react-native-windows.git"
+      commit="$CommitId$" />
+    <tags>react react-native react-native-windows microsoft c c++ cpp c# csharp reunion projectreunion</tags>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <description>React Native for Windows for Project Reunion apps</description>
+    <dependencies>
+      <group targetFramework=".NETCoreApp3.0" />
+      <group targetFramework="UAP10.0" />
+      <group targetFramework=".NETFramework4.6" />
+      <group targetFramework="net5.0">
+        <dependency id="Microsoft.Windows.CsWinRT" version="1.3.0" exclude="Build,Analyzers" />
```

Comment:
Reunion packages have been for the most part leaving out the CsWinRT reference as dependency here as the dependencies from CsWinRT (WinRT.Runtime) also come from the TFM for consumers.

Reply:
A lot of this stuff is coming from the cswinrt walkthrough: https://docs.microsoft.com/en-us/windows/uwp/csharp-winrt/net-projection-from-cppwinrt-component

Can we maybe update the walkthrough if the steps are no longer accurate?

Reply:
filed https://github.com/microsoft/CsWinRT/issues/895 
fyi @angelazhangmsft